qemu/tests/qemu_guest_agent.py

```diff
@@ -137,6 +137,7 @@ def _check_ga_pkg(self, session, cmd_check_pkg):
                     qga_v = re.findall(pattern, pkg, re.I)
                     if qga_v:
                         version_list.append(qga_v[0])
+                self.qga_v = version_list[1]
```

Comment:
```suggestion
                self.qga_v = version_list[-1]
```
Otherwise you might hit exception.

Reply:
okay, thanks